server/src/app.ts

```diff
@@ -47,6 +58,8 @@ const identityClient =
 app.get('/api/config', configController(config));
 app.get('/api/token', tokenController(identityClient, config));
 
+app.post('/api/createSurveyResult', storeSurveyResult);
+
```

Comment:
should util\CosmosClient instance be created here once and injected into controller classes wherever needed?

Reply:
This is interesting idea. I am thinking that this layer is more router layer that not sure if we want to pass `cosmosClient` instance and pass it to certain controller methods

Reply:
This class is the middleware layer. The middleware layer usually takes care of creating the instances which are supposed to be kept alive for the lifetime of the application. And then takes care of passing these to the classes where required.

You would want the db layer to be instantiated only once in the application as it's an expensive operation.